overlord/fdestate/fdestate.go

```diff
@@ -218,7 +220,12 @@ func initializeState(st *state.State) error {
 
 	// FIXME mount points will be different in recovery or factory-reset modes
 	// either inspect degraded.json, or use boot.HostUbuntuDataForMode()
-	dataUUID, dataErr := disksDMCryptUUIDFromMountPoint(dirs.SnapdStateDir(dirs.GlobalRootDir))
+	dataDir := dirs.GlobalRootDir
```

Comment:
What makes / the correct mount point to look at to find the data disk on classic? maybe a small comment is needed to explain this.